webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -225,6 +223,30 @@ public List<AtlasEvaluatePolicyResponse> evaluatePolicies(List<AtlasEvaluatePoli
         return response;
     }
 
+    /**
+     * API to get accessors info such as roles/groups/users who can perform specific action
+     */
+    @POST
```

Comment:
This API will be open to all logged users, should we add authorization for  this may be next phase.
 `https://beta-atlas.atlan.com/api/atlas/v2/entity/accessors`API ? 
since its exposing secured user policy details.